inc/fields.class.php

```diff
@@ -329,4 +328,39 @@ public static function updateVisibility($currentValues) {
 
       return $questionToShow;
    }
+
+   /**
+    * checks if a field type exists
+    *
+    * @param string $type type of field to test for existence
+    * @return boolean
+    */
+   public static function getFieldClassname($type) {
+      return 'PluginFormcreator' . ucfirst($type) . 'Field';
+   }
+
+   /**
+    * checks if a field type exists
+    *
+    * @param string $type type of field to test for existence
+    * @return boolean
+    */
+   public static function fieldTypeExists($type) {
+      return (class_exists(self::getFieldClassname($type)));
```

Comment:
Adding something like `&& is_a(self::getFieldClassname($type), PluginFormcreatorFieldInterface::class, true)` could make code more defensive.

Reply:
I would also say that 
```php
is_a(self::getFieldClassname($type), PluginFormcreatorFieldInterface::class, true)
``` 
is sufficient by itself to check that the class exists.

Reply:
Indeed.